apps/meteor/client/providers/OmnichannelProvider.tsx

```diff
@@ -41,6 +50,18 @@ const OmnichannelProvider: FC = ({ children }) => {
 
 	const accessible = hasAccess && omniChannelEnabled;
 	const iceServersSetting: any = useSetting('WebRTC_Servers');
+	const isEnterprise = useHasLicenseModule('livechat-enterprise') === true;
+
+	const getPriorities = useEndpoint('GET', '/v1/livechat/priorities');
+
+	const {
+		data: { priorities = [] } = {},
+		isInitialLoading: isLoadingPriorities,
+		isError: isErrorPriorities,
+	} = useQuery(['/v1/livechat/priorities'], () => getPriorities({ sort: JSON.stringify({ sortItem: 1 }) }), {
+		staleTime: millisecondsToMinutes(10),
```

Comment:
Wouldn’t be better to use a stream instead of invalidating every 10 minutes? We even had the PR for the backend part of the streams here: https://github.com/RocketChat/Rocket.Chat/pull/27291